src/main/java/org/broadinstitute/consent/http/models/support/SupportTicketCreator.java

```diff
@@ -1,115 +0,0 @@
-package org.broadinstitute.consent.http.models.support;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-import org.broadinstitute.consent.http.configurations.ServicesConfiguration;
-import org.broadinstitute.consent.http.db.InstitutionDAO;
-import org.broadinstitute.consent.http.db.UserDAO;
-import org.broadinstitute.consent.http.enumeration.SupportRequestType;
```

Comment:
Can `SupportRequestType` be removed too? It looks like the only uses of it are to create support tickets

Reply:
We need this for the next phase of work